lib/ansible/plugins/strategy/free.py

```diff
@@ -234,6 +234,7 @@ def run(self, iterator, play_context):
 
             if len(included_files) > 0:
                 all_blocks = dict((host, []) for host in hosts_left)
+                failed_includes_hosts = set()
```

Comment:
if we ever get different messages per host (I cannot see that currently happen) this should be a dict instead to record those messages individually